packages/mc-scripts/src/utils/credentialsStorage.js

```diff
@@ -0,0 +1,54 @@
+const { existsSync, writeFileSync, readFileSync, mkdirSync } = require('fs');
```

Comment:
Before we forget: please kebab case filenames. :)

Reply:
Ah!.. yes. Thanks, @tdeekens